analyzers/its/AllSonarAnalyzerRules.ruleset

```diff
@@ -3956,7 +3956,7 @@
     <Rule Id="S3946" Action="Warning" />
     <Rule Id="S3947" Action="Warning" />
     <Rule Id="S3948" Action="Warning" />
-    <Rule Id="S3949" Action="Warning" />
+    <Rule Id="S3949" Action="None" />
```

Comment:
please add a comment here and line 13961 with a link to the ticket to re-implement the rule

Reply:
Done. The issue can be found here: https://github.com/SonarSource/sonar-dotnet/issues/4631